contributing_to_docs/doc_guidelines.adoc

```diff
@@ -63,6 +63,25 @@ If using section anchors, they must be all lowercase letters, with no line space
 ----
 
 == Links, hyperlinks, and cross references
+Whenever possible use inline embedded links to other topics or other references.
+
+Example of recommended usage where possible:
+
+----
+Rollbacks revert part of an application back to a previous deployment. Rollbacks can be performed using the REST API or
+the link:cli.html#deployment-rollbacks[OpenShift CLI].
+----
+
+This will render as so:
+
+Rollbacks revert part of an application back to a previous deployment. Rollbacks can be performed using the REST API or the link:cli.html#deployment-rollbacks[OpenShift CLI].
+
+As compared to this, which is not recommended:
+
+----
+Rollbacks revert an application back to a previous deployment and can be performed using the REST API or
+the OpenShift CLI. See the link:cli.html#deployment-rollbacks[CLI documentation] for more details.
```

Comment:
@nhr I see where the confusion may be, and my bad for not clarifying. The examples don't show the method or markup of "creating" the link. What I was trying to show was to avoid linking by creating a separate sentence that begins with something like "See [this] for more information on x". Hopefully I've clarified it with my updates.